specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerEffectiveConfiguration.json

```diff
@@ -124,6 +120,77 @@
       },
       "description": "The network manager effective configuration"
     },
+    "SecurityRuleCollection": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SecurityRuleCollectionPropertiesFormat",
+          "description": "Indicates the properties for the network manager rule collection."
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system metadata related to this resource.",
+          "$ref": "./network.json#/definitions/SystemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Defines the rule collection."
+    },
+    "SecurityRuleCollectionPropertiesFormat": {
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "A display name of the rule collection."
+        },
+        "description": {
+          "type": "string",
+          "description": "A description of the rule collection."
+        },
+        "securityType": {
```

Comment:
By associating securityType to a single SecurityRuleCollection - Are we limiting for RuleCollection to be either AdminRules or User rules, but not both?

Reply:
Yes